packages/ibm-products/src/components/Card/Card.tsx

```diff
@@ -21,68 +21,122 @@ import { CardFooter } from './CardFooter';
 import { pkg } from '../../settings';
 const componentName = 'Card';
 
-// Default values for props
-const defaults = {
-  actionIcons: Object.freeze([]),
-  actionsPlacement: 'bottom',
-  clickZone: 'one',
-  mediaPosition: 'top',
-  overflowActions: Object.freeze([]),
-  primaryButtonKind: 'primary',
-  primaryButtonPlacement: 'bottom',
-  productive: false,
-  secondaryButtonKind: 'secondary',
-  secondaryButtonPlacement: 'bottom',
-  titleSize: 'default',
+type ActionIcon = {
+  id?: string;
+  icon?: () => ReactNode;
+  onKeydown?: () => void;
+  onClick?: () => void;
+  iconDescription?: string;
+  href?: string;
 };
+type OverflowActions = {
+  id?: string;
+  itemText?: string;
+  onClick?: () => void;
+  onKeydown?: () => void;
+};
+
+type Metadata = {
+  id?: string;
+  icon?: () => ReactNode;
+  iconDescription?: string;
+  onClick?: () => void;
+  href?: string;
+};
+interface CardProp extends PropsWithChildren {
+  actionIcons?: readonly ActionIcon[];
+  actionsPlacement?: 'top' | 'bottom';
+  children?: ReactNode;
+  className?: string;
+  clickZone?: 'one' | 'two' | 'three';
+  description?: string | ReactNode;
+  disabled?: boolean;
+  footerActionIcon?: React.ElementType;
+  getStarted?: boolean;
+  label?: string | ReactNode;
+  media?: ReactNode;
+  mediaPosition?: 'top' | 'left';
+  metadata?: readonly Metadata[];
+  onClick?: () => void;
+  onKeyDown?: () => void;
+  onPrimaryButtonClick?: () => void;
+  onSecondaryButtonClick?: () => void;
+  overflowActions?: readonly OverflowActions[];
+  overflowAriaLabel?: string;
+  pictogram?: () => ReactNode;
+  primaryButtonDisabled?: boolean;
+  primaryButtonHref?: string;
+  primaryButtonIcon?: React.ElementType;
+  primaryButtonKind?: 'primary' | 'ghost';
+  primaryButtonPlacement?: 'top' | 'bottom';
+  primaryButtonText?: string;
+  productive?: boolean;
+  secondaryButtonDisabled?: boolean;
+  secondaryButtonHref?: string;
+  secondaryButtonIcon?: React.ElementType;
+  secondaryButtonKind?: 'secondary' | 'ghost';
+  secondaryButtonPlacement?: 'top' | 'bottom';
+  secondaryButtonText?: string;
+  sequence?: number;
+
+  /**
+   * **Experimental?** For all cases a `Slug` component can be provided.
+   * Clickable tiles only accept a boolean value of true and display a hollow slug.
+   */
+  slug?: ReactNode | boolean;
+
+  status?: 'complete' | 'incomplete';
+  title?: string | ReactNode;
```

Comment:
```suggestion
  title?: ReactNode;
```
ReactNode supports string, no need to add it here